pkg/controller/generate-registry/group.go

```diff
@@ -248,7 +248,10 @@ func (c *Controller) group(logE *logrus.Entry, pkgName string, releases []*Relea
 
 	for _, group := range groups {
 		release := group.releases[0]
-		pkgInfo := &registry.PackageInfo{}
+		pkgInfo := &registry.PackageInfo{
+			RepoOwner: pkgInfo.RepoOwner,
+			RepoName:  pkgInfo.RepoName,
+		}
```

Comment:
This and carrying the pkgInfo around is not too nice at all, but I needed some way to get the repo owner and name down to patchRelease. Figured I'd start simple, this appears to work at least for some cases.